lib/kino_benchee.ex

```diff
@@ -1,2 +1,5 @@
 defmodule KinoBenchee do
```

Comment:
@josevalim since it's under `livebook-dev`, do we want `Kino.Benchee`? At this point there are no explicit APIs, but I imagine we could have some functions for specific visualizations in the future.

Reply:
I am thinking we should just remove this module altogether. Docs can go in the README and we can use the README or the guide as the starting page for ExDoc.

Reply:
Yeah, we could also use `guides/components.livemd` as the starting page. I guess my question was more hypothetical :D

Reply:
I went ahead and deleted the unused module. Adding a components.livemd like is done here: https://hexdocs.pm/kino_vega_lite/components.html